freud/box.pyx

```diff
@@ -779,7 +779,7 @@ cdef class ParticleBuffer:
             dereference(self.thisptr.getBufferParticles().get()).size()
         if not buffer_size:
             return np.array([[]], dtype=np.uint32)
-        cdef unsigned int[::1] buffer_ids = \
+        cdef const unsigned int[::1] buffer_ids = \
```

Comment:
I'm not sure if it's correct to use `const` in cases like this. The tests still pass, but this is kind of an output array...?